### PR TITLE
Prep for 0.10.0 release

### PR DESCRIPTION
**Pull request by StephenWeatherford**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,43 @@
...
+- Insert item... (variables, parameters, resources, functions, outputs) **implemented by Nils Hedström @nilshedstrom, thanks!**
+Look for this in the editor context menu or the ARM Template Outline view!
+- Completions for resourceId arguments based on resources found inside the template
+- New `Application Gateway` and `Application Gateway and Firewall` snippets (thanks Emmanuel Auffray @ManuInNZ!)
+
+### Changed
+
+- Suggestion: Auto-complete should not insert "()" [#501](https://github.com/microsoft/vscode-azurearmtools/issues/501)
+- Removed `arm-param-value` snippet in parameter files (replaced by new parameter auto-completions)
+- Add lightbulbs to increase discoverability of parameter/variable renames
```
Comment (nilshedstrom) [inline CHANGELOG.md:24]: Maybe include a "thanks"?

Comment (StephenWeatherford) [reply CHANGELOG.md:24]: Ugh, missed that, sorry!  Will add...

Comment (StephenWeatherford) [reply CHANGELOG.md:24]: Ugh, missed that, sorry!  Will add...

Comment (StephenWeatherford) [reply CHANGELOG.md:24]: https://github.com/microsoft/vscode-azurearmtools/pull/703  

I didn't miss anything else, did I?

Comment (nilshedstrom) [reply CHANGELOG.md:24]: As far as I know you did not miss anything else :)

Comment (StephenWeatherford) [reply CHANGELOG.md:24]: Good.  Thanks for letting us know.

**Pull request merged**